From 6848f26cfdfb39b58ce32b4f9782fc265129fea6 Mon Sep 17 00:00:00 2001
From: wh <wanghang@rock-chips.com>
Date: Wed, 30 Sep 2020 18:01:31 +0800
Subject: [PATCH 203/297] add deskclock to allow-in-power-save attributes

android 8.0, we found deskclock can not alarm if enter background,
for example: reboot or exit deskclock app then wait some miniute.

is also happen in Q and R

Signed-off-by: wh <wanghang@rock-chips.com>
Change-Id: I1c4bce75845ff9f6a41fbfa840d90e2380584f8c
---
 data/etc/platform.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/etc/platform.xml b/data/etc/platform.xml
index cf312165dd9..2848910f0b6 100644
--- a/data/etc/platform.xml
+++ b/data/etc/platform.xml
@@ -252,6 +252,7 @@
     <allow-in-power-save package="com.android.cellbroadcastreceiver.module" />
     <allow-in-power-save package="com.android.cellbroadcastreceiver" />
     <allow-in-power-save package="com.android.shell" />
+    <allow-in-power-save package="com.android.deskclock" />
 
     <!-- Whitelist system providers -->
     <allow-in-power-save-except-idle package="com.android.providers.calendar" />
-- 
2.35.1

